hr_accident_at_work/models/hr_accident_at_work.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+from odoo import fields, models, api
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT as OE_DFORMAT
+from datetime import datetime
+
+
+class HrAccidentatwork(models.Model):
+    _name = 'hr.accident.at.work'
+    _description = 'HR Accident at work'
+
+    employee_id = fields.Many2one(comodel_name='hr.employee', required=True,
+                                  string="Employee")
+    accident_address = fields.Text(string='Accident Adress')
+    accident_date = fields.Datetime(string='Date', required=True)
+    accident_sequence = fields.Selection(selection=[
+        ('EXCEPTIONAL FREQUENCY', 'EXCEPTIONAL FREQUENCY'),
+        ('REGULAR FREQUENCY', 'REGULAR FREQUENCY'),
+    ], string='Frequency Type')
+    accident_department = fields.Integer(string='Department', required=True)
+    accident_location = fields.Char(string='City')
+    number_of_employee_concerned = fields.Integer(
+        string='Number of Employee Concerned', default=1, required=True)
+    number_of_employee_concerned_acknowledge_by_social_security = \
+        fields.Integer(string='Number of Employee concerned and acknowledge by'
+                              ' the Social Security Authority', default=0)
+    social_security_authority_decision = fields.Text(
+        string='Social Security Authority Decision')
+    accident_almost = fields.Boolean(string='Almost accident ?', default=False)
+    accident_during_commute = fields.Boolean(string='Accident during the '
+                                                    'commuting ?',
+                                             default=False)
+    accident_circumstances = fields.Char(string='circumstances', required=True)
+    wound_description = fields.Text(string='Wound description', required=True)
+    online_wound_declaration_done = fields.Boolean(
+        string='Online Wound Declaration Done?', default=False)
```

Comment:
Why specifically "online"? This may not be a requirement, and sounds too specific to your case.